test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -421,7 +429,13 @@ def forward(self, x):
 
         m1 = self._get_conv_bn_model(has_conv_bias=False)
         m2 = M2(self.conv_class, self.bn_class)
-        example_inputs = (torch.randn(3, 3, 5, 5),)
+
+        assert self.dim in [1, 2]
+        if self.dim == 1:
+            example_inputs = (torch.randn(3, 3, 5),)
```

Comment:
nit: can you store these in the test case: `self.conv_dim_to_example_inputs = {1: ..., 2: ...}`?

Reply:
I actually already store these in `self.example_inputs` for most tests. The ones you see here are specific to the test case, which creates its custom test models, so I don't think we can do that generally

Reply:
I see, why do we need these that's specific to test case? is it because of the args in conv/bn? is it possible to just align everything?

Reply:
It's because in some tests we need custom models (e.g. with padding, stride, or extra ops). The same example inputs don't seem to work in these cases so it's easier to just use new example inputs

Reply:
oh I see, OK